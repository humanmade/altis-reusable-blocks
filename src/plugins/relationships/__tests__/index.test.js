import { name, settings } from '../';

jest.mock( '../containers/Relationships', () => () => null );

describe( 'name', () => {

	test( 'is as expected', () => {
		expect( name ).toBe( 'altis-reusable-block-relationships' );
	} );

} );

describe( 'settings', () => {

	test( 'are as expected', () => {
		expect( settings ).toMatchSnapshot();
	} );

} );
