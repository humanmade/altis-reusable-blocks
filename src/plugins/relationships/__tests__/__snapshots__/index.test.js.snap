// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`settings are as expected 1`] = `
Object {
  "icon": "admin-links",
  "render": [Function],
}
`;
